node/utils/utils.ts

```diff
@@ -89,3 +87,22 @@ export const convertGlovoProductsToCompare = (
 
   return items
 }
+
+export const isValidAffiliateId = (
+  affiliateId: string,
+  stores: StoreInfo[]
+): boolean => {
+  if (Number(affiliateId)) {
+    return false
+  }
+
+  const storeIsConfigured = stores.some(
+    (store) => store.affiliateId === affiliateId
+  )
+
+  if (!storeIsConfigured) {
+    return false
+  }
+
+  return true
```

Comment:
A possible small simplification, `.some` returns a boolean. You can return it directly and avoid the `if` statement.